tests/cpu/ops/fixedfunction/FixedFunctionOpCPU_tests.cpp

```diff
@@ -265,6 +265,138 @@ OCIO_ADD_TEST(FixedFunctionOpCPU, aces_dark_to_dim_10)
                            1e-7f,
                            __LINE__);
     }
+}
+
+OCIO_ADD_TEST(FixedFunctionOpCPU, aces_gamut_map_13)
```

Comment:
This is awesome!!!

Is there a URL you could add as a comment regarding the source of the data?

Reply:
Sure thing, will do now.

Reply:
Sure thing, will do now.